### DISCUSSION TITLE
GitHub Desktop. Automatic branch change

### DISCUSSION THREAD

**Discussion by AlekzzR** [community/community#46726] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Mistake. Description: There are many branches in the repository. I click "Merge into current branch ...". The calculation is performed in relation to the "master" branch, if at this moment I select the branch that I actually want to merge into the current one, then nothing will happen. But the error is not in this, but in the fact that if you wait until the calculation is performed by the wizard, then click the desired branch, the branch is selected, information about conflicts is displayed and after a time the branch automatically changes to another. Now I was ready to click Merge, but the branch changed and almost merge another branch. And this automatic shift is very dangerous.

**Comment by steveward** [comment]
@AlekzzR thanks for sharing. Would you mind opening an issue in https://github.com/desktop/desktop/issues so we can dig into this further? I wasn't able to reproduce this with a quick test, but there may be something we're missing.
